ted_sws/notice_fetcher/services/notice_fetcher.py

```diff
@@ -5,10 +5,14 @@
 from ted_sws.domain.model.manifestation import XMLManifestation
 from ted_sws.domain.model.metadata import TEDMetadata
 from ted_sws.domain.model.notice import Notice
-from ted_sws.notice_fetcher.adapters.ted_api import TedDocumentSearch
+from ted_sws.notice_fetcher.adapters.ted_api_abc import DocumentSearchABC
 
 
 class NoticeFetcherABC(abc.ABC):
+    """
```

Comment:
If we decide to have only functions in the service layer of the application, then the class NoticeFetcher class can be merged into DocumetnSearch class. This is to be discussed.